src/Kernels/Plasticity.cpp

```diff
@@ -172,7 +171,7 @@ namespace seissol::kernels {
       adjKrnl.execute();
 
       // calculate plastic strain with first dof only (for now)
```

Comment:
this comment is outdated

Reply:
Ok, changed it to `// calculate plastic strain`